services/ui-src/src/containers/HomeState.js

```diff
@@ -32,34 +56,30 @@ const HomeState = () => {
       <ul className="quarterly-items">
         {years[year].quarters.map(element => {
           return (
-            <li key={`${state}-${year}-${element}`}>
-              <Link href={`/forms/${state}/${years[year].year}/${element}`}>
+            <li key={`${state}-${years[year].year}-${element}`}>
+              <Link href={`/forms/${state}/${years[year].year}./${element}`}>
```

Comment:
I think the . before /${element} needs to be removed as it's generating the URL looking like this: http://localhost:3000/forms//2021./1